src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -661,10 +675,34 @@ private enum ClusterDatabaseChangeType
             ValueChanged
         }
 
-        public void UnloadDirectly(StringSegment databaseName, [CallerMemberName] string caller = null)
+        public void UnloadDirectly(StringSegment databaseName, long? wakeup = null, [CallerMemberName] string caller = null)
         {
             LastRecentlyUsed.TryRemove(databaseName, out _);
-            DatabasesCache.RemoveLockAndReturn(databaseName, CompleteDatabaseUnloading, out var _, caller).Dispose();
+            DatabasesCache.RemoveLockAndReturn(databaseName, CompleteDatabaseUnloading, out _, caller).Dispose();
+            ScheduleDatabaseWakeup(databaseName.Value, wakeup);
+        }
+
+        private void ScheduleDatabaseWakeup(string name, long? wakeup)
+        {
+            if (name == null)
+                return;
+
+            if (_wakeupTimers.TryRemove(name, out var timer))
+            {
+                timer.Dispose();
+            }
+
+            if(wakeup.HasValue == false || wakeup.Value == long.MaxValue)
+                return;
+            
+            var dueTime = Math.Max(1000, (int)TimeSpan.FromTicks(wakeup.Value - DateTime.Now.Ticks).TotalMilliseconds);
```

Comment:
I think it should be `SystemTime.UtcNow` not `DateTime.Now` ?

Reply:
No, backups are explicitly local time.